runtime/sema/type_tags.go

```diff
@@ -218,6 +218,7 @@ const (
 	transactionTypeMask
 
 	invalidTypeMask
+	capabilityControllerTypeMask
```

Comment:
Doesn't really matter much, but maybe but this after `transactionTypeMask` (before the gap) – it looks like the gap is there to separate the invalid type mask from the rest. (Not sure, @SupunS might know)